client/components/deposits-overview/test/index.tsx

```diff
@@ -187,6 +197,22 @@ describe( 'Deposits Overview information', () => {
 		expect( getByText( 'Estimated' ) ).toBeTruthy();
 		expect( getByText( '—' ) ).toBeTruthy();
 	} );
+
+	test( 'Confirm new account waiting period notice does not show', () => {
+		global.wcpaySettings.accountStatus.deposits.completed_waiting_period = true;
+		const { getByText } = render( <DepositsOverview /> );
+		expect(
+			getByText( 'Your first deposit is held for seven business days' )
+		).toBeFalsy();
+	} );
+
+	test( 'Confirm new account waiting period notice shows', () => {
+		global.wcpaySettings.accountStatus.deposits.completed_waiting_period = false;
+		const { getByText } = render( <DepositsOverview /> );
+		expect(
+			getByText( 'Your first deposit is held for seven business days' )
+		).toBeTruthy();
```

Comment:
This one is failing because a `.a11y-speak-region` is being added with a duplicate of this text. `getByText` [expects exactly 1 element to match and will fail the test otherwise](https://testing-library.com/docs/queries/about) (this means we also don't have to call `toBeTruthy()`). We can tell `getByText` to `ignore` a selector.

Also, the string you're searching for is rendered as `Your first deposit is held for seven business days.  Why? (opens in a new tab) `, so using regex pattern syntax `/` rather than `'` will allow a partial match.
```suggestion
		const { getByText } = render( <DepositsOverview /> );
		getByText( /Your first deposit is held for seven business days/, {
			ignore: '.a11y-speak-region',
		} );
```